### PR TITLE
ci: add reusable daily workflow

### PR DESCRIPTION
**Pull request by helenakallekleiv**

### CODE REVIEW

**Review by hknutsen — COMMENTED** [review]
Referring to the `inputs` context will fail when this workflow is triggered by the schedule.

To solve this, you need to create a separate reusable workflow (with only the `workflow_call` trigger), and then another workflow that calls that reusable workflow (with the `schedule` trigger).

**Comment by helenakallekleiv** [comment]
> Referring to the `inputs` context will fail when this workflow is triggered by the schedule.
> 
> To solve this, you need to create a separate reusable workflow (with only the `workflow_call` trigger), and then another workflow that calls that reusable workflow (with the `schedule` trigger).

Noted, any naming suggestions for the separate workflow?

**Review by hknutsen — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -13,11 +28,11 @@ jobs:
...
+          days-before-stale: ${{ inputs.days-before-stale}}
+          days-before-close: ${{ inputs.days-before-close}}
+          stale-issue-label: ${{ inputs.label}}
+          stale-issue-message: There has been no activity on this issue for ${{ inputs.days-before-stale}} days. ${{ inputs.label}} label will be added. If no additional activity occurs, the issue will be closed in ${{ inputs.days-before-close}} days.
+          close-issue-message: There has been no activity on this stale issue for ${{ inputs.days-before-close}} days. This issue will now be closed. If new activity occurs, the issue will reopen.
+          stale-pr-label: ${{ inputs.label}}
+          stale-pr-message: There has been no activity on this pull request for ${{ inputs.days-before-stale}} days. ${{ inputs.label}} label will be added. If no additional activity occurs, the pull request will be closed in ${{ inputs.days-before-close}} days.
+          close-pr-message: There has been no activity on this stale pull request for ${{ inputs.days-before-close}} days. This pull request will now be closed. If new activity occurs, the pull request will reopen.
```
Comment (sebastianlolv) [inline .github/workflows/stale.yml:38]: Ekstremt pirkete, sorry 😅 

```suggestion
          days-before-stale: ${{ inputs.days-before-stale }}
          days-before-close: ${{ inputs.days-before-close }}
          stale-issue-label: ${{ inputs.label }}
          stale-issue-message: There has been no activity on this issue for ${{ inputs.days-before-stale }} days. ${{ inputs.label }} label will be added. If no additional activity occurs, the issue will be closed in ${{ inputs.days-before-close }} days.
          close-issue-message: There has been no activity on this stale issue for ${{ inputs.days-before-close }} days. This issue will now be closed. If new activity occurs, the issue will reopen.
          stale-pr-label: ${{ inputs.label }}
          stale-pr-message: There has been no activity on this pull request for ${{ inputs.days-before-stale }} days. ${{ inputs.label }} label will be added. If no additional activity occurs, the pull request will be closed in ${{ inputs.days-before-close }} days.
          close-pr-message: There has been no activity on this stale pull request for ${{ inputs.days-before-close }} days. This pull request will now be closed. If new activity occurs, the pull request will reopen.
```

Comment (helenakallekleiv) [reply .github/workflows/stale.yml:38]: Love it 🙏🏼

**Review by sebastianlolv — APPROVED** [review]

**Review by hknutsen — APPROVED** [review]

**Pull request merged**